packages/client/ui/react-ui/src/hooks/useAuth.ts

```diff
@@ -0,0 +1,15 @@
+import { WalletContext } from "@/providers/CrossmintAuthProvider";
+import { useContext } from "react";
+
+import { useAuth as useAuthCore } from "@crossmint/client-sdk-auth-core";
+
+export function useAuth() {
+    const walletContext = useContext(WalletContext);
+
+    if (!walletContext) {
+        throw new Error("useAuth must be used within an WalletProvider");
```

Comment:
Should this say "... within a CrossmintAuthProvider" instead, since that's what's exposed to external devs?